server/container/guice/jpa-guice/pom.xml

```diff
@@ -194,7 +194,7 @@
                  <dependency>
                     <groupId>org.apache.derby</groupId>
                     <artifactId>derby</artifactId>
-                    <scope>test</scope>
+                     <scope>compile</scope>
```

Comment:
I'm not sure about this modification, you are embedding Derby for every builds of jpa-guice; but the user may use another database.
Derby should come in another docker container (such as Cassandra).